tests/test_processors.py

```diff
@@ -850,12 +844,17 @@ def __init__(self):
 
         logger = structlog.stdlib.get_logger()
 
+        # These are different when running under async
+        exclude_keys = ["thread", "thread_name"]
+
         callsite_params = self.get_callsite_parameters()
         await logger.info("baz")
+        logger_params = json.loads(string_io.getvalue())
 
-        assert {"event": "baz", **callsite_params} == json.loads(
-            string_io.getvalue()
-        )
+        [callsite_params.pop(key) for key in exclude_keys]
+        [logger_params.pop(key) for key in exclude_keys]
```

Comment:
That seems unnecessarily clever:

```suggestion
        # These are different when running under async
        for key in ["thread", "thread_name"]:
            callsite_params.pop(key)
            logger_params.pop(key)
```

Reply:
Thought it was neater if that list ever had to grow.